src/dbnode/storage/series/reader_test.go

```diff
@@ -640,25 +648,7 @@ func TestReaderReadEncodedRobust(t *testing.T) {
 						break
 					}
 				} else {
-					// If the data was not in the disk cache then expect that and setup a query
```

Comment:
since disk reads are now lazy (after series and cache), this needed to be moved down.